### ISSUE TITLE
Creating a Croatian lemmatizer - licence issue

### ISSUE THREAD

**Issue by isaric** [explosion/spaCy#4164]
Hello all,

I am looking to extend the croatian language model in spaCy with a look-up lemmatizer.
I found a great source of lemmas (over 100 000 lemmas in more than a million forms) here : http://meta-share.ffzg.hr/repository/browse/croatian-morphological-lexicon-v50/2d429672703d11e28a985ef2e4e6c59e27b37c59b92d42a5be839f7daff7ecfb/

I have already adaptet the lexicon into a lemmatizer and tested it out locally. It seems to work well.
The issue I`m having with submitting this lemmatizer to the spaCy codebase is the lexicons licence:

The lexicon is licensed under CC BY NC SA licence and spaCy is released under an MIT licence.

TL;DR; Is it okay to adapt the Croatian morphological lexicon into a lemmatizer given the different licenses that the two projects use?

**Comment by isaric** [comment]
@ines I submitted this issue to you over LinkedIn and Gitter about a week ago but I haven`t gotten any feedback so I am resubmitting it here

**Comment by Pavle992** [comment]
Hi @isaric , I also found this source where you have even more lemmas for Croatian.
[Croatian lemmas](https://www.clarin.si/repository/xmlui/handle/11356/1232)
Not sure about the licences though.

**Comment by ines** [comment]
Yes, this is a much better place to ask that question than on LinkedIn (never read my messages there, sorry!).

> The lexicon is licensed under CC BY NC SA licence and spaCy is released under an MIT licence.

CC BY NC is not going to work unfortunately 😞 It's not compatible with spaCy's commercial-friendly MIT license. We'd also be reluctant at this point to build models trained on CC BY NC data, because it'd mean that the models could only be available for non-commercial use, which is really unattractive for most of our users.

**Comment by Pavle992** [comment]
> Yes, this is a much better place to ask that question than on LinkedIn (never read my messages there, sorry!).
> 
> > The lexicon is licensed under CC BY NC SA licence and spaCy is released under an MIT licence.
> 
> CC BY NC is not going to work unfortunately  It's not compatible with spaCy's commercial-friendly MIT license. We'd also be reluctant at this point to build models trained on CC BY NC data, because it'd mean that the models could only be available for non-commercial use, which is really unattractive for most of our users.

I had the same question @ines ,  so I assume the **CC BY-SA** should work?

**Comment by ines** [comment]
> so I assume the CC BY-SA should work?

We'd be okay with publishing models trained on CC BY-SA data, but the license is not compatible with MIT. So code or other resources with that license couldn't be included in the library and/or this repo.

**Comment by isaric** [comment]
@Pavle992  the source you mentioned is licensed under GPL v3 according to https://reldi.spur.uzh.ch/blog/croatian-lexicon/
@ines Is GPL v3 ok to use in the spaCy library?
